project/plugins.sbt

```diff
@@ -1,9 +1,8 @@
-val sbtTypelevelVersion = "0.4.22"
+val sbtTypelevelVersion = "0.5.0-RC7"
```

Comment:
Why don't release 0.5.0 since we are going to migrate the flagman one project to it?

Reply:
Well, it's arbitrary :) Personally, I would like to migrate more "flagship" projects such as Cats Effect, FS2, http4s and Skunk so I can be fully convinced that everything is working correctly for a final release.